R/testing.R

```diff
@@ -206,6 +206,19 @@ build_git_demo <- function() {
 
   git_checkout_branch("master", root = path)
 
+  # On windows, make ".git" folder non-hidden,
```

Comment:
It might be nice to isolate the ick a bit by putting it into it's own function - it would just take `path` as an argument

Reply:
Try now...